src/customFields.gs

```diff
@@ -46,44 +43,11 @@ function getCustomFields( format ) {
  * @return {Array}    Array of custom Jira Fields
  */
 function fetchCustomFields() {
-  var method = "field", _customFieldsRaw = [], customFields = [];
-
-  var ok = function(respData, httpResp, status) {
-    if(respData) {
-      debug.log("Response of fetchCustomFields(); respData: %s", respData);
-
-      // reset custom epic field
-      UserStorage.setValue('jst_epic', fieldEpic);
-
-      var arrSupportedTypes = ['string', 'number', 'datetime', 'date', 'option', 'array|option', 'array|string', 'user', 'array|user', 'group', 'array|group', 'version', 'array|version'];
-
-      // add data to export
-      _customFieldsRaw.push.apply(_customFieldsRaw, respData.map(function(cField) {
-        var _type = (cField.schema ? cField.schema.type : null) || null;
-        if(cField.schema && cField.schema.items) {
-          _type += '|' + cField.schema.items;
-        }
-
-        // EPIC customization
-        if (cField.schema && cField.schema.custom) {
-          if (cField.schema.custom.indexOf(':gh-epic-link') > -1) {
-            fieldEpic.link_key = cField.key || cField.id;
-          }
-          if (cField.schema.custom.indexOf(':gh-epic-label') > -1) {
-            fieldEpic.label_key = cField.key || cField.id;
-          }
-        }
-
-        return {
-          key:        cField.key || cField.id, // Server API returns ".id" only while Cloud returns both with same value
-          name:       cField.name,
-          custom:     cField.custom,
-          schemaType: _type,
-          supported:  (arrSupportedTypes.indexOf(_type) > -1)
-        };
-      }) )
```

Comment:
This is the big chunk of code that was duplicated in getAllJiraFields in jiraCommon